modules/post-installation-configuring-nutanix-failure-domains.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/adding-nutanix-failure-domains.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="post-installation-configuring-nutanix-failure-domains_{context}"]
+= Adding failure domains to the Infrastructure CR
+
+You add failure domains to an existing Nutanix cluster by modifying its Infrastructure custom resource definition (CR) (`infrastructures.config.openshift.io`).
```

Comment:
Remove "definition". Should be "its Infrastructure custom resource (CR)".

Reply:
Done.